docs/02-app/01-building-your-application/01-routing/13-route-handlers.mdx

```diff
@@ -115,36 +115,28 @@ The following examples show how to combine Route Handlers with other Next.js API
 
 ### Revalidating Cached Data
 
-You can [revalidate cached data](/docs/app/building-your-application/data-fetching/caching-and-revalidating#revalidating-data) using the [`next.revalidate`](/docs/app/building-your-application/data-fetching/caching-and-revalidating#revalidating-data) option:
+You can [revalidate cached data](/docs/app/building-your-application/data-fetching/incremental-static-regeneration) using Incremental Static Regeneration (ISR):
+
+```ts filename="app/posts/route.ts" switcher
+export const revalidate = 60
 
-```ts filename="app/items/route.ts" switcher
 export async function GET() {
-  const res = await fetch('https://data.mongodb-api.com/...', {
-    next: { revalidate: 60 }, // Revalidate every 60 seconds
-  })
-  const data = await res.json()
+  let data = await fetch('https://api.vercel.app/blog')
```

Comment:
might blocking, but having `const` would be nice to avoid users extending the example but result in errors by mistake